README.md

```diff
@@ -545,10 +547,10 @@ about that build. The build for the tag is where the actual deployment to npm wi
 one of the scripts doesn't do exactly what you need, then copy it directly into your project, modify it, and update
 the Travis config to run you modified copy instead.*
 
-### Bitbucket Server / TeamCity
-You can now configure `pr-bumper` to work with something other than Travis CI and GitHub.
-The only other configuration that has been tested is TeamCity and Bitbucket Server.
-Example TeamCity and Bitbucket Server configuration:
+### Bitbucket / Bitbucket Server / TeamCity
+You can now configure `pr-bumper` to work with something other than Travis CI, GitHub and Bamboo.
+The only other configuration that has been tested is TeamCity, Bitbucket and Bitbucket Server.
+Example TeamCity, Bitbucket and Bitbucket Server configuration:
```

Comment:
I'm not sure about these edits, it makes it sound like Travis, GitHub, and Bamboo are all in same workflow. Similarly, TeamCity, Bitbucket, and Bitbucket Server. 

I think that example `.pr-bumper.json` is still specifically for TeamCity and Bitbucket Server. If you want to add another section that gives an example of Bitbucket and Bamboo, that'd be great, but I don't think we want to confuse this example by including two VCS systems or two CI systems.